internal/painter/gl/gl_common.go

```diff
@@ -15,6 +15,12 @@ import (
 	"fyne.io/fyne/v2/internal/painter"
 )
 
+// Texture represents an uploaded GL texture
+type Texture cache.TextureType
+
+// NoTexture is the zero value for a Texture
+var NoTexture = Texture(cache.NoTexture)
```

Comment:
Not sure why this needed to be aligned with the `cache.NoTexture` - it was just a zero value before

Reply:
With GL Wasm it is a JS value and can not be compared to zero easily anymore.

Reply:
The addition of `Valid` seems to remove the need for this?

Reply:
We still need the cache.NoTexture, I will check if NoTexture in the gl package need to be exposed or not, but it certainly need to exist as cache.NoTexture need to be used to initialize the default value properly as with wasm it get initialized to undefined or something like that otherwise.

Reply:
I checked I properly did switch the gl NoTexture to gl.noTexture and stopped exporting it. So I think we should be good with what is exported and what is not.